docs/osquery/osquery.asciidoc

```diff
@@ -121,9 +121,12 @@ image::images/scheduled-pack.png[Shows queries in the pack and details about eac
 
 [float]
 [[osquery-manage-query]]
-== Edit saved queries
+== Saved queries
 
-Add or edit saved queries from the *Saved queries* tab.
+You can save queries after running a live query or from the *Saved queries* tab. 
+Once you save a query, you can only edit it from the *Saved queries* area.
+
+How to add or edit saved queries from the *Saved queries* tab:
```

Comment:
Not sure that this sentence is necessary.

Reply:
Great input throughout, thanks @gchaps. This is the only comment I didn't put an update in for. There may be a better way to handle it, but I'm trying to make it clear here that these steps are about how to add or edit saved queries specifically from the **Saved queries** tab. We could make it a sub-heading of this section, but "Add or edit saved queries from the *Saved queries* tab" feels too wordy for that. I'm leaning towards leaving this as is, but I'm open to suggestions if you have any thoughts on a better way to handle it.

Reply:
Let's take out the word "how" and go with this:

> To add or edit saved queries from the **Saved queries** tab:

Reply:
That works! I just pushed the update.